packages/yoga/src/Dropdown/native/Options.android.jsx

```diff
@@ -46,31 +46,30 @@ const OptionText = styled.Text`
 `;
 
 const Options = ({ options, selectedOption, onSelect }) => (
-  <ScrollView contentContainerStyle={{ width: '100%', height: 60 * 3.6 }}>
```

Comment:
We don't need this `ScrollView` anymore? Has the behaviour changed or we didn't need it at all?

Reply:
We don't because its children is a FlatList Component.

The FlatList renders only the visible elements (kinda VirtualList thing) and the ScrollView will always render everything. Putting it in that order, FlatList as a child of ScrollView, the React Native will fire a warning saying this:

```
VirtualizedLists should never be nested inside plain ScrollViews with the same orientation
 - use another VirtualizedList-backed container instead
```

So, I removed that ScrollView, and everything keeps working as it should. ✨